infra/terraform/xania.tf

```diff
@@ -187,7 +187,7 @@ resource "aws_iam_role_policy_attachment" "xania_attach_policy" {
 }
 
 resource "aws_instance" "XaniaNode" {
-  ami = "ami-03bd6c18a631bc24a"
+  ami = "ami-04505e74c0741db8d"
```

Comment:
Just bringing it up to date. The amd 20.04 image in us-east-1,  https://cloud-images.ubuntu.com/locator/ec2/

Reply:
will applying this change recreate the AMI though? (we'll have to ensure everything's backed up and restored if so)

Reply:
I'll undo it -- for the moment, we actually want the option to be able to make other infra changes (e.g. networking) by rerunning terraform, and without forcing a change to the  base image.